Cargo.toml

```diff
@@ -30,8 +30,8 @@ singlepass = ["cosmwasm-vm/default-singlepass"]
 cranelift = ["cosmwasm-vm/default-cranelift"]
 
 [dependencies]
-cosmwasm-std = { git = "https://github.com/cosmwasm/cosmwasm", rev = "v0.9.0-alpha1", features = ["iterator"]}
-cosmwasm-vm = { git = "https://github.com/cosmwasm/cosmwasm", rev = "v0.9.0-alpha1", features = ["iterator"] }
+cosmwasm-std = { git = "https://github.com/cosmwasm/cosmwasm", rev = "65df0f0c602f851790a3e7014ec20523f83a3dbf", features = ["iterator"]}
+cosmwasm-vm = { git = "https://github.com/cosmwasm/cosmwasm", rev = "65df0f0c602f851790a3e7014ec20523f83a3dbf", features = ["iterator"] }
```

Comment:
Can you use `v0.9.0-alpha2` tagged right now?

Reply:
Just did that (1 minute ago). Thanks for the tag :+1: